scripts/release_info.py

```diff
@@ -51,16 +56,29 @@
                     'CURRENT_COLLECTION_SCHEMA_VERSION']
 FIRST_OPPIA_COMMIT = '6a7138f5f603375e58d1dc3e1c4f1c80a126e249'
 NO_LABEL_CHANGELOG_CATEGORY = 'Uncategorized'
+# This should match the indentation of span and li elements used
+# in credits section of
+# core/templates/dev/head/pages/about-page/about-page.directive.html.
+SPAN_INDENT = '              '
+LI_INDENT = '                '
+# This line should match the line after div element for credits in
+# about-page.directive.html.
+LINE_AFTER_Z_CREDITS = (
+    '            <p translate="I18N_ABOUT_PAGE_CREDITS_'
+    'TAB_TEXT_BOTTOM" translate-values="{listOfNames: $ctrl.listOfNames}">\n')
+ABOUT_PAGE_FILEPATH = (
```

Comment:
Should generally use os.path.join for platform-independence.

Reply:
Done.